patterns/ecr-pull-through-cache/main.tf

```diff
@@ -11,10 +8,13 @@ terraform {
     }
     helm = {
       source  = "hashicorp/helm"
-      version = ">= 2.13"
+      version = ">= 2.9"
```

Comment:
why downgrading the helm version?

Reply:
Not downgrading, it's just compatible with 2.9 or higher to match other patterns.